angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome-modal-template.html

```diff
@@ -0,0 +1,12 @@
+<div class="welcome-popup center">
+  <h1 ng-bind-html="welcome.title"></h1>
+  <img src="{{ welcome.img }}" class="hidden-xs">
+	<div class="sr-only" tabindex="0" id="welcomeContent" aria-labelledby="welcomeContent">
```

Comment:
This `aria-labelledby="welcomeContent"` looks weird to me, in that the div marked as labelledby welcomeContent itself has the `id` welcomeContent, so isn't that saying it's labeled by itself?

Is there something that should be done to the `<img` tag itself to make it accessible?  Maybe that's the element that should have the `aria-labelledby` attribute on it?  And maybe it's more of an [`aria-describedby`](http://www.w3.org/WAI/GL/wiki/Using_aria-describedby_to_provide_descriptions_of_images), that is, more of a description than a label?

The technique here feels kind of like externalizing the text description of the image, what with it being designated `sr-only`.  As in, here's an image, and here's some text that describes that image that you might want to read alternatively to that image.  Potential in going further an externalizing it via the [`longdesc` technique](http://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/H45.html)?